smt/applications/mixed_integer.py

```diff
@@ -224,9 +224,9 @@ def __init__(
             )
             and self._surrogate.options["categorical_kernel"] is None
         ):
-            self._surrogate.options["categorical_kernel"] = (
-                MixIntKernelType.HOMO_HSPHERE
-            )
+            self._surrogate.options[
+                "categorical_kernel"
+            ] = MixIntKernelType.HOMO_HSPHERE
```

Comment:
Why is this reformatted? Do you still use black?

Reply:
no, ruff format

Reply:
ruff has been updated to 3.6 last week and to 3.7 this week.

Reply:
ruff has been updated to 3.6 last week and to 3.7 this week.